src/helpers/likeAndDislike.js

```diff
@@ -0,0 +1,24 @@
+export const fromDisLikeToLike = (likesUsers, dislikeUsers, id) => {
+  dislikeUsers.forEach((userId, index) => {
+    if (userId === id) {
+      dislikeUsers.splice(index, 1);
+      likesUsers.push(id);
+    }
+  });
+  return likesUsers;
+};
+
+export const fromLikeToDislike = (usersDislike, usersLikes, id) => {
+  const updatedUsers = [];
+  usersLikes.forEach((uId, i) => {
+    if (uId === id) {
+      usersLikes.splice(i, 1);
+      usersDislike.push(id);
```

Comment:
This is causing O(n^2) and it is very bad, why not change it into O(n).

Reply:
I see it bad but I can't find any other way I can use to solve that problem, If you have any you can share to me